src/components/content/home/DeployDashBoard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { Alert, Card, Col, Row, Statistic } from 'antd';
+import { Link } from 'react-router-dom';
+import { myServicesRoute } from '../../utils/constants';
+import { useQuery } from '@tanstack/react-query';
+import { ApiError, Response, ServiceService, ServiceVo } from '../../../xpanse-api/generated';
+import { convertStringArrayToUnorderedList } from '../../utils/generateUnorderedList';
+
+export function DeployDashBoard(): React.JSX.Element {
+    const [successfulDeployments, setSuccessfulDeployments] = useState<number>(0);
+    const [failedDeployments, setFailedDeployments] = useState<number>(0);
+    const [successfulDestroys, setSuccessfulDestroys] = useState<number>(0);
+    const [failedDestroys, setFailedDstroys] = useState<number>(0);
+    const [servicesLoadingError, setServicesLoadingError] = useState<React.JSX.Element>(<></>);
+    const listDeployedServicesQuery = useQuery({
+        queryKey: ['listDeployedServices'],
+        queryFn: () => ServiceService.listDeployedServices(undefined, undefined, undefined, undefined, undefined),
+        refetchOnWindowFocus: false,
+    });
+
+    useEffect(() => {
+        let successfulDeploymentsCount: number = 0;
+        let failedDeploymentsCount: number = 0;
+        let successfulDestroysCount: number = 0;
+        let failedDestroysCount: number = 0;
+        if (listDeployedServicesQuery.isSuccess && listDeployedServicesQuery.data.length > 0) {
+            listDeployedServicesQuery.data.forEach((serviceItem: ServiceVo) => {
+                if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_SUCCESSFUL) {
+                    successfulDeploymentsCount++;
+                }
+                if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_FAILED) {
+                    failedDeploymentsCount++;
+                }
+                if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_SUCCESSFUL) {
+                    successfulDestroysCount++;
+                }
+                if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_FAILED) {
+                    failedDestroysCount++;
+                }
+            });
+        }
+        setSuccessfulDeployments(successfulDeploymentsCount);
+        setFailedDeployments(failedDeploymentsCount);
+        setSuccessfulDestroys(successfulDestroysCount);
+        setFailedDstroys(failedDestroysCount);
+    }, [listDeployedServicesQuery.data, listDeployedServicesQuery.isSuccess]);
+
+    useEffect(() => {
```

Comment:
same here. no useEffect needed. Directly return the failure alert when isError is true.